core/lib/tee_verifier/src/lib.rs

```diff
@@ -253,6 +253,7 @@ impl TeeVerifierInput {
             (false, TreeLogEntry::ReadMissingKey { .. }) => TreeInstruction::Read(key),
             _ => {
                 tracing::error!("Failed to map LogQuery to TreeInstruction");
+                tracing::error!("{:?} {:?}", log_query.rw_flag, *tree_log_entry);
```

Comment:
nitpick: it would probably fir better directly to the line above - also this log is not temporary so we might forget it here.. I'll approve PR to avoid blocking you, but please resolve in one of the follow-ups